SPECS/kubevirt/kubevirt.spec

```diff
@@ -0,0 +1,1670 @@
+#
+# spec file for package kubevirt
+#
+# Copyright (c) 2022 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+
+Summary:        Container native virtualization
+Name:           kubevirt
+Version:        0.51.0
+Release:        1%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System/Management
+URL:            https://github.com/kubevirt/kubevirt
+#Source0:       https://github.com/kubevirt/kubevirt/archive/refs/tags/v0.51.0.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+Source1:        disks-images-provider.yaml
+%global debug_package %{nil}
```

Comment:
Please move this to the top of the file - we try to keep the macros there as much as possible.

Reply:
Done